src/components/Login.js

```diff
@@ -14,25 +14,22 @@ export default function Login() {
     const history = useHistory()
 
     // asyc function that handles submit button
-    async function handleSubmit(e) {
+     function handleSubmit(e) {
         // prevent form from refreshing 
         e.preventDefault()
-        try {
```

Comment:
There are no more error messages since the try catches are gone. The user will not know what went wrong if we get rid of the error messages. ex: wrong password -> page just resets with no explanation

Reply:
We will need to create a bug for this issue. Error messages can be added to the chaining in `.finally()`

Reply:
Bug created at #70